app/templates/contacts/overview.hbs

```diff
@@ -22,11 +22,23 @@
       </Auk::Toolbar::Item>
     </Auk::Toolbar::Group >
     <Auk::Toolbar::Group  @position="right">
-      <Auk::Button @icon="add" @skin="primary" @route="contacts.new">
+      <Auk::Button @icon="add" @skin="primary" @route="contacts.new" class="auk-u-mr-2">
         Verzendlijst aanmaken
       </Auk::Button>
+      <Auk::Button @icon="add" @skin="primary" {{on "click" this.openContactItemModal}}>
+        Contactpersoon aanmaken
+      </Auk::Button>
```

Comment:
We kunnen deze knoppen best conditioneel maken op basis van de geselecteerd tab zodanig dat 'Verzendlijsten aanmaken' getoond wordt op de 'Verzendlijsten'-tab en 'Contactpersoon aanmaken' enkel bij 'Contactpersonen'-tab.

Je kan hiervoor de `router` service injecteren en gebruiken maken van de [`currentRoute` property](https://api.emberjs.com/ember/release/classes/RouterService/properties/currentRoute?anchor=currentRoute).